clar2wasm/src/wasm_generator.rs

```diff
@@ -1696,7 +1696,7 @@ impl WasmGenerator {
                     builder.call(
                         self.module
                             .funcs
-                            .by_name("stdlib.is_in_regtest")
+                            .by_name("is_in_regtest")
```

Comment:
Sames as above. `stdlib.` prefix is now part of function present on Clarity standard library.